### ISSUE TITLE
pipenv update fails with version 6.0.2

### ISSUE THREAD

**Issue by ewdurbin** [kennethreitz/pipenv#491]
Pipfile:

```
[[source]]
url = "https://pypi.python.org/simple"
verify_ssl = true

[packages]
requests = "*"
```

Steps to reproduce:

```
pipenv --rm
pip install --user pipenv==5.4.2
pipenv lock
pipenv install
pipenv update
pipenv update
pip install --user pipenv==6.0.2
pipenv update
pipenv update
pipenv lock
pipenv update
```

💥 

specific error message:

```
In --require-hashes mode, all requirements must have their versions pinned with ==. These do not:
    chardet<3.1.0,>=3.0.2 from https://pypi.python.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl#md5=0004b00caff7bb543a1d0d0bd0185a03 (from requests==2.18.4->-r /var/folders/0l/srl335_n5q959lz0bmxpn_v80000gn/T/pipenv-WrWIvv-requirement.txt (line 1))
    certifi>=2017.4.17 from https://pypi.python.org/packages/40/66/06130724e8205fc8c105db7edb92871c7fff7d31324d7f4405c762624a43/certifi-2017.7.27.1-py2.py3-none-any.whl#md5=c78c8e571e3556b8f0222bd221e471ac (from requests==2.18.4->-r /var/folders/0l/srl335_n5q959lz0bmxpn_v80000gn/T/pipenv-WrWIvv-requirement.txt (line 1))
    urllib3<1.23,>=1.21.1 from https://pypi.python.org/packages/63/cb/6965947c13a94236f6d4b8223e21beb4d576dc72e8130bd7880f600839b8/urllib3-1.22-py2.py3-none-any.whl#md5=1c11e1c80371cc4e89911071010a98d1 (from requests==2.18.4->-r /var/folders/0l/srl335_n5q959lz0bmxpn_v80000gn/T/pipenv-WrWIvv-requirement.txt (line 1))
```

Resulting Pipfile.lock:

```
{
    "_meta": {
        "hash": {
            "sha256": "da2810af0c3b5333e0de2fce9bea2a228812e2014e5f5fe3b1c533badc6c24e4"
        },
        "host-environment-markers": {
            "implementation_name": "cpython",
            "implementation_version": "0",
            "os_name": "posix",
            "platform_machine": "x86_64",
            "platform_python_implementation": "CPython",
            "platform_release": "17.0.0",
            "platform_system": "Darwin",
            "platform_version": "Darwin Kernel Version 17.0.0: Thu Aug 24 22:01:05 PDT 2017; root:xnu-4570.1.46~3/RELEASE_X86_64",
            "python_full_version": "2.7.13",
            "python_version": "2.7",
            "sys_platform": "darwin"
        },
        "pipfile-spec": 2,
        "requires": {},
        "sources": [
            {
                "url": "https://pypi.python.org/simple",
                "verify_ssl": true
            }
        ]
    },
    "default": {
        "certifi": {
            "hashes": [
                "sha256:54a07c09c586b0e4c619f02a5e94e36619da8e2b053e20f594348c0611803704",
                "sha256:40523d2efb60523e113b44602298f0960e900388cf3bb6043f645cf57ea9e3f5"
            ],
            "version": "==2017.7.27.1"
        },
        "chardet": {
            "hashes": [
                "sha256:fc323ffcaeaed0e0a02bf4d117757b98aed530d9ed4531e3e15460124c106691",
                "sha256:84ab92ed1c4d4f16916e05906b6b75a6c0fb5db821cc65e70cbd64a3e2a5eaae"
            ],
            "version": "==3.0.4"
        },
        "idna": {
            "hashes": [
                "sha256:8c7309c718f94b3a625cb648ace320157ad16ff131ae0af362c9f21b80ef6ec4",
                "sha256:2c6a5de3089009e3da7c5dde64a141dbc8551d5b7f6cf4ed7c2568d0cc520a8f"
            ],
            "version": "==2.6"
        },
        "requests": {
            "hashes": [
                "sha256:6a1b267aa90cac58ac3a765d067950e7dbbf75b1da07e895d1f594193a40a38b",
                "sha256:9c443e7324ba5b85070c4a818ade28bfabedf16ea10206da1132edaa6dda237e"
            ],
            "version": "==2.18.4"
        },
        "urllib3": {
            "hashes": [
                "sha256:06330f386d6e4b195fbfc736b297f58c5a892e4440e54d294d7004e3a9bbea1b",
                "sha256:cc44da8e1145637334317feebd728bd869a35285b93cbb4cca2577da7e62db4f"
            ],
            "version": "==1.22"
        }
    },
    "develop": {}
}
```

**Comment by kennethreitz** [comment]
Thanks for the report.

**Comment by kennethreitz** [comment]
Boy that was a big one. Fixed in v6.0.3.
